samples/msal-node-samples/silent-flow/test/silent-flow-aad-agc-public.spec.ts

```diff
@@ -134,6 +134,9 @@ describe("Silent Flow AAD AGC Public Tests", () => {
             await NodeCacheTestUtils.expireAccessTokens(TEST_CACHE_LOCATION);
             tokens = await NodeCacheTestUtils.waitForTokens(TEST_CACHE_LOCATION, 2000);
             const expiredAccessToken = tokens.accessTokens[0];
+            
+            // Wait to ensure new token has new iat
+            await new Promise(r => setTimeout(r, 1000));
```

Comment:
Do you want to define this as a constant somewhere?  We use 1k in a few places.  The 2k on line 135 could also be a const.

Reply:
Done